core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,45 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var answerGroups = responsesService.getAnswerGroups();
+        // First push all choice indices into an array
+        var choiceArray = [];
+        for (var i = 0; i < choices; i++) {
+          choiceArray.push(i);
+        }
+        /* Then cycle through answerGroup rule_specs
+        and put all of them into an array */
+        var answerArray = [];
+        for (var j = 0; j < answerGroups.length; j++) {
+          /* Make sure to check if a an answerGroup rule_spec
```

Comment:
I don't think you need two cases here. You could just iterate over all the rules in rule_specs.